lya/lya.py

```diff
@@ -65,8 +65,11 @@ def __init__(self, *argz, **kwz):
 	def __setitem__(self, k, v):
 		super(AttrDict, self).__setitem__(k, self.map_types(v))
 	def __getattr__(self, k):
-		if not (k.startswith('__') or k.startswith('_OrderedDict__')): return self[k]
-		else: return super(AttrDict, self).__getattr__(k)
+		try:
+			if not (k.startswith('__') or k.startswith('_OrderedDict__')): return self[k]
+			else: return super(AttrDict, self).__getattr__(k)
+		except KeyError:
+			raise AttributeError
```

Comment:
for clarity, i propose to use `AttributeError(k)`

it might be necessary to contain the try/except to the get-item branch to let unrelated exceptions bubble up correctly